src/pages/RegisterAndLogin/Components/Inputs.tsx

```diff
@@ -50,16 +53,30 @@ const Inputs = ({ page }: { page: string }) => {
   };
 
   const submitHandler = () => {
-    console.log("email: ", formData.email);
-    console.log("password: ", formData.password);
-
-    toast({
-      title: "Account created.",
-      description: "We've created your account for you.",
-      status: "success",
-      duration: 5000,
-      isClosable: true,
-    });
+    Axios.post("/user/register", formData)
+      .then((res) => {
+        console.log(res);
+        toast({
+          title: "Account created.",
+          description: "We've created your account for you.",
+          position: "top-right",
+          status: "success",
+          duration: 5000,
```

Comment:
as you can see here, we use two times duration, position isClosable properties. Find a solution so that we don't repeat ourselves